includes/compat/class-wc-stripe-email-customer-failed-refund.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Failed Refund Notification (shopper)
+ *
+ * @since 9.6.0
+ */
+class WC_Stripe_Email_Customer_Failed_Refund extends WC_Stripe_Email_Failed_Refund {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->id          = 'failed_refund_customer';
```

Comment:
As above RE avoiding collisions.
```suggestion
		$this->id          = 'wc_stripe_failed_refund_customer';
```